### DISCUSSION TITLE
Does Vorta support multiple devices at once?

### DISCUSSION THREAD

**Discussion by smnhfr** [borgbase/vorta#1544] [FAQ] [answered]
Hey,
A few days ago I set a VirtualBox (Windows 10) and I wanted to back up the files with Vorta. I already use Vorta on my Main Computer (Fedora 37) and I was wondering if I can use 2 devices at once to back up without problems. Before trying it out and braking something with my archives, I wanted to ask here, for being safe.

Thanks in advance

Bye, Simon

**Comment by m3nu** [comment, marked as answer]
You can point 2 devices to the same repo, but they can't use it at the same time. Else no issue.
